gdc_client/download/parser.py

```diff
@@ -75,7 +82,48 @@ def download(parser, args):
         ids.add(i['id'])
 
     client = get_client(args, args.token_file)
-    client.download_files(ids)
+    downloaded_files, errors = client.download_files(ids)
+
+    if args.retry_amount > 0:
+        files_not_downloaded = []
+
+        for uuid, _ in errors.items():
```

Comment:
Haha yeah I guess you're right. I think I was using the values before but scrapped it. Good eye.